pcsx2/gui/Dialogs/CreateMemoryCardDialog.cpp

```diff
@@ -252,8 +252,12 @@ void Dialogs::CreateMemoryCardDialog::CreateControls()
 		.	SetToolTip(_t("Use at your own risk.  Erratic memory card behavior is possible (though unlikely)."))
 		.	SetInt(64),
 
-		RadioPanelItem(_("Folder [experimental]"), _("Store memory card contents in the host filesystem instead of a file."))
-		.	SetToolTip(_t("Automatically manages memory card contents so that the console only sees files related to the currently running software. Allows you to drag-and-drop files in and out of the memory card with your standard file explorer. This is still experimental, so use at your own risk!"))
+		RadioPanelItem(_("Folder [best with PCSX2]"), _("Transforms the memory card into an usable folder with files inside."))
+		.	SetToolTip(_t("Gives every game 8 MB for the saves (essentially a whole memcard).\n" 
```

Comment:
I suppose this is sort of accurate but it can be confusing; it may imply that it's allocating 8 MB on the host system for each game, which is not the case. I would maybe consider saying something like "Stores memory card contents in a folder. Only exposes the save files for the running game, rather than the entire memory card".